parse_date.re

```diff
@@ -1239,7 +1240,7 @@ weekdayof        = (reltextnumber|reltexttext) space (dayfull|dayabbr) space 'of
 		}
 
 		if (*ptr != '\0') {
-			s->time->z = timelib_parse_zone((char **) &ptr, &s->time->dst, s->time, &tz_not_found, s->tzdb, tz_get_wrapper);
+			s->time->z = timelib_parse_zone((char **) &ptr, &s->time->dst, s->time, 0, &tz_not_found, s->tzdb, tz_get_wrapper);
```

Comment:
I don't like using arbitrary numbers (`0` and `1` further down) as flags. Please define a macro value for them, or what is even better, define two (static) functions, one for each flag value, which then can share the common implementation.